scripts/README.md

```diff
@@ -0,0 +1,63 @@
+# Development scripts for Kubernetes documentation
+
```

Comment:
I'd like to see a section here listing each script with a description of what it does.